lib/inspec/backend.rb

```diff
@@ -54,9 +54,18 @@ def self.create(config)
       end
 
       cls = Class.new do
+        attr_accessor :cache_resources
         include Base
 
+        def initialize
+          @cache_resources = false
+        end
+
         define_method :backend do
+          if @cache_resources
```

Comment:
Is there another place we can/should put the `enable_cache` calls? The way I read this, every time something in InSpec calls for the `backend` it will send these methods to the Train connection. That could happen a LOT.

Maybe we just need an enable/disable method on the backend itself that can be called once when appropriate?

Also, we're making an assumption here that caching is disabled by default in Train, which is half correct -- we always cache files, and don't provide a way to disable currently. Should this have an `else` statement where we disable the cache?

This probably warrants further discussion after standup... until Train is consistent with the default behavior of both files and commands, we may want to consider only manipulating the command cache for now and then deal with files later.